velox/functions/prestosql/tests/ProbabilityTest.cpp

```diff
@@ -221,5 +221,30 @@ TEST_F(ProbabilityTest, invBetaCDF) {
   VELOX_ASSERT_THROW(invBetaCDF(3, 5, 1.1), "p must be in the interval [0, 1]");
 }
 
+TEST_F(ProbabilityTest, gammaCDF) {
+  const auto gammaCDF = [&](std::optional<double> shape,
+                            std::optional<double> scale,
+                            std::optional<double> value) {
+    return evaluateOnce<double>("gamma_cdf(c0, c1, c2)", shape, scale, value);
+  };
+
+  EXPECT_DOUBLE_EQ(0.96675918913720599, gammaCDF(0.5, 3.0, 6.8).value());
```

Comment:
Please add tests for values std::nullopt, kNan, kDoubleMax, kDoubleMin along with kInf

Reply:
Thanks @aditi-pandit .
Added tests for values std::nullopt, kNan, kDoubleMax, kDoubleMin.